content/locales/en.ts

```diff
@@ -41,22 +41,100 @@ export const en = {
     title: "Personal information",
     breadcrumb: "Profile | new",
     info: "Please fill in these information to create your profile.",
+    namePlaceholder: "Enter your full name",
+    usernamePlaceholder: "Enter your username",
+    profilePlaceholder: "Enter your profile URL",
+    websitePlaceholder: "Enter your website URL",
+    picturePlaceholder: "Enter your picture URL",
+    zoneInfoPlaceholder: "Enter your zone info",
+    localePlaceholder: "Enter your language",
+    nameLabel: "Full name *",
+    usernameLabel: "Username",
+    birthDateLabel: "Birth date",
+    profileLabel: "Profile URL",
+    websiteLabel: "Website URL",
+    pictureLabel: "Picture URL",
+    zoneInfoLabel: "Zone info",
+    localeLabel: "Language",
+    add: "Add my information",
+    update: "Update my information",
+    cancel: "Cancel",
   },
   "/account/profile/user-profile/edit": {
     title: "Personal information",
     breadcrumb: "Profile | edit",
+    namePlaceholder: "Enter your full name",
+    usernamePlaceholder: "Enter your username",
+    profilePlaceholder: "Enter your profile URL",
+    websitePlaceholder: "Enter your website URL",
+    picturePlaceholder: "Enter your picture URL",
+    zoneInfoPlaceholder: "Enter your zone info",
+    localePlaceholder: "Enter your language",
+    nameLabel: "Full name *",
+    usernameLabel: "Username",
+    birthDateLabel: "Birth date",
+    profileLabel: "Profile URL",
+    websiteLabel: "Website URL",
+    pictureLabel: "Picture URL",
+    zoneInfoLabel: "Zone info",
+    localeLabel: "Language",
+    add: "Add my information",
+    update: "Update my information",
+    cancel: "Cancel",
   },
   "/account/profile/addresses/[id]": {
     title: "Personal information",
     breadcrumb: "Address",
+    primary: "Primary",
+    update: "Update this address",
+    delete: "Delete this address",
+    mark: "Use this address as my main address",
+    infoMark: "You are about to set this address as main.",
+    infoDelete: "You are about to delete this address.",
+    deleteButton: "Delete",
+    keep: "Keep this address",
+    confirm: "Confirm",
+    cancel: "Cancel",
   },
   "/account/profile/addresses/[id]/edit": {
     title: "Personal information",
     breadcrumb: "Address | edit",
+    streetAddressPlaceholder: "Enter your street address",
```

Comment:
Now I think about it, and maybe it's not for this PR, but I don't think we should use placeholders as instructions (e.g: `"Enter you name"`) but as examples instead: `"e.g: John Rambo"`, because we already have the labels for that.

Do you see what I mean?

From MDN (https://www.w3schools.com/tags/att_input_placeholder.asp):
> The placeholder attribute specifies a short hint that describes the expected value of an input field (e.g. a sample value or a short description of the expected format).

Reply:
Yeah, maybe I could just switch the label with the placeholder? All in all, I'll make a new task for this 👍 .

Reply:
Yep, new task is the way to go, so we can discuss it during the refinement 🙂